### ISSUE TITLE
finding CoC

### ISSUE THREAD

**Issue by shannonpileggi** [rladies/rladies.github.io#144]
is the CoC navigable from the main site?

**Comment by drmowinckels** [comment]
yes, its in the footer. Now, if you cannot find it, it means its placed poorly. Im open for suggestions!

**Comment by drmowinckels** [comment]
@shannonpileggi is this still an issue or something we should improve somehow?
